### DISCUSSION TITLE
what is mining?

### DISCUSSION THREAD

**Discussion by leo-brown244** [Cyfrin/foundry-full-course-cu#2010] [General]
i want to know more detail...

**Comment by EngrPips** [comment]
Hello @leo-brown244, You should start here `https://www.bankrate.com/investing/what-is-bitcoin-mining/`

**Comment by CadetTalha** [comment]
Hey @leo-brown244 
Take a look how mining actually works in a simple way 

Suppose that you are a miner (a person using specific software for this task )
Two parties A and B  just do some type of transaction on blockchain, now the miner verified that transaction and locked it into a block(immutable ,we can just read and add something but cannot delete or update )

For doing that task, you will get paid as a mining reward 
( paid by that blockchain in their native token just like bitcoin paid their miners in btc )+some sort of fee that these two parties had paid)

In simple, mining is a mechanism where miner verified the txn and locked it into a block and get paid for that
